system-test/system_test.sh

```diff
@@ -10,11 +10,15 @@ set -eox pipefail
 
 cd $(dirname $0)
 
+# The list of tested versions below should be in sync with node's
+# official releases. https://nodejs.org/en/about/releases/
 if [[ -z "$BINARY_HOST" ]]; then
   ADDITIONAL_PACKAGES="python3 g++ make"
-  NODE_VERSIONS=(10 12 14 15 16 node)
+  NODE_VERSIONS=(12 14 16 17 node)
```

Comment:
Is it worthwhile to add Node 18 here?

Eventually, "node" will reference Node 20, so adding 18 ensures we don't unexpectedly stop testing Node 18.

Reply:
I don't have strong opinions on this so I'll explicitly add 18 to cover the case you mentioned. I'll also keep "node" so that any breakages in 20 can be detected early.